src/follower.py

```diff
@@ -73,35 +73,60 @@ def stop(self):
 
     """
     Turn the wheels towards the last recognized object.
+    Pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
     """
-    Converts the camera's angle scale to the same scale as the wheels.
+    Follows the tag by panning camera towards the same direction as the wheels.
```

Comment:
That's not what this is doing at all. It's not turning it towards the wheels. It's turning it left or right at 15 degrees (that's a different problem)>